src/js/tabs/motors.js

```diff
@@ -269,10 +270,11 @@ motors.initialize = async function (callback) {
             mixer:              FC.MIXER_CONFIG.mixer,
             reverseMotorSwitch: FC.MIXER_CONFIG.reverseMotorDir,
             escprotocol:        FC.PID_ADVANCED_CONFIG.fast_pwm_protocol + 1,
-            feature3:           FC.FEATURE_CONFIG.features.isEnabled('MOTOR_STOP'),
+            feature4:           FC.FEATURE_CONFIG.features.isEnabled('MOTOR_STOP'),
```

Comment:
This should be checked - as I checked the code very recent to verify assignment of featureX and I'm sure feature3 is MOTOR_STOP ===> feature = features.bit - 1

Reply:
Hmm, it should be 4 indeed. Having trouble with 27 getting enabled.

Reply:
Duh, needs ESC configured on ports tab.

You were correct. See https://github.com/betaflight/betaflight-configurator/pull/3382

Reply:
https://github.com/betaflight/betaflight-configurator/pull/3382